irohad/torii/impl/command_service.cpp

```diff
@@ -207,7 +214,7 @@ namespace torii {
 
     log_->debug("StatusStream waiting finish, hash: {}", request_hash->hex());
 
-    if (not *finished) {
+    if (not(*finished)) {
```

Comment:
remove redundant braces plz, otherwise it highlighted as a function
same with the following